test/integration/src/main/java/io/pravega/test/integration/compatibility/CompatibilityChecker.java

```diff
@@ -224,8 +224,13 @@ private void checkStreamDelete() {
     }
 
     public static void main(String[] args) throws DeleteScopeFailedException {
+        String uri = System.getProperty("controllerUri");
+        if (uri == null) {
+            log.error("Enter correct gradle with correct controller URI. i.e : ./gradlew compatibilityCheck -Du=\"tcp://localhost:9090\"");
+            System.exit(1);
```

Comment:
Should this `main` method execute `System.exit(0)` after completing its execution?

Reply:
done